compiler/src/iree/compiler/Codegen/Common/Passes.td

```diff
@@ -497,6 +497,18 @@ def TileAndDistributeToWorkgroupsPass :
   ];
 }
 
+def TileAndDistributeToWorkgroupsUsingForallOpPass :
+    InterfacePass<"iree-codegen-tile-and-distribute-to-workgroups-using-forall-op",
+                  "mlir::FunctionOpInterface"> {
+  let summary = "Tile and distribute operation to workgroups (using scf.forall op)";
+  let dependentDialects = [
+    "affine::AffineDialect",
+    "IREE::LinalgExt::IREELinalgExtDialect",
+    "scf::SCFDialect",
+    "tensor::TensorDialect",
```

Comment:
+`IREECodegenDialect` for the workgroup mapping attributes. Lowering configs can come from different dialects.